src/utils/date-string-regexp.ts

```diff
@@ -1 +1,2 @@
-export const dateStringRegExp = /^(\d{4})([-./])(\d{2})(\2)(\d{2})$/
+export const dateStringRegExp =
```

Comment:
can you add a comment clarifying the regex (that it matches `dd-...` and `yy-...`)